Networking/Services/SessionManager.swift

```diff
@@ -112,14 +111,18 @@ public protocol SessionManagerType {
  */
 final public class SessionManager: SessionManagerType {
 
-    fileprivate let _keychain: KeychainSwift
+    fileprivate let _keychainService: KeychainServiceType
     fileprivate var _currentUserFetcher: CurrentUserFetcherType?
     
     fileprivate let _sessionToken = MutableProperty<String?>(.none)
     fileprivate let _currentUser = MutableProperty<AuthenticableUser?>(.none)
     
-    public init(keychain: KeychainSwift = KeychainSwift()) {
-        _keychain = keychain
+    convenience public init() {
```

Comment:
Because if I made `init(keychain: KeychainSwift = KeychainSwift())` public, I also had to make  `KeychainSwift` public, and I didn't want to. It's just an internal class created to test `SessionManager` providing a mock.